src/configs/all.ts

```diff
@@ -0,0 +1,236 @@
+            ["Function", "Avoid using the `Function` type. Prefer a specific function type, like `() => void`."],
+            ["Boolean", "Avoid using the `Boolean` type. Did you mean `boolean`?"],
+            ["Number", "Avoid using the `Number` type. Did you mean `number`?"],
+            ["String", "Avoid using the `String` type. Did you mean `string`?"],
+            ["Symbol", "Avoid using the `Symbol` type. Did you mean `symbol`?"],
+        ],
+    },
+    "member-access": [true, "check-accessor", "check-constructor"],
+    "member-ordering": [true, {
+        "order": "statics-first",
+        "alphabetize": true,
+    }],
+    "no-any": true,
+    "no-empty-interface": true,
+    "no-import-side-effect": true,
+    // Technically this is not the strictest setting, but don't want to conflict with "typedef"
+    "no-inferrable-types": [true, "ignore-params"],
+    "no-internal-module": true,
+    "no-magic-numbers": true,
+    "no-namespace": true,
+    "no-non-null-assertion": true,
+    "no-reference": true,
+    "no-var-requires": true,
+    "only-arrow-functions": true,
+    "prefer-for-of": true,
+    "promise-function-async": true,
+    "typedef": [
+        true,
+        "call-signature",
+        "arrow-call-signature",
+        "parameter",
+        "arrow-parameter",
+        "property-declaration",
+        "variable-declaration",
+        "member-variable-declaration",
+    ],
+    "typedef-whitespace": [true, {
+        "call-signature": "nospace",
+        "index-signature": "nospace",
+        "parameter": "nospace",
+        "property-declaration": "nospace",
+        "variable-declaration": "nospace",
+    }, {
+        "call-signature": "onespace",
+        "index-signature": "onespace",
+        "parameter": "onespace",
+        "property-declaration": "onespace",
+        "variable-declaration": "onespace",
+    }],
+    "unified-signatures": true,
+
+    // Functionality
+    "await-promise": true,
+    // "ban": no sensible default
+    "curly": true,
+    "forin": true,
+    // "import-blacklist": no sensible default
+    "label-position": true,
+    "no-arg": true,
+    "no-bitwise": true,
+    "no-conditional-assignment": true,
+    "no-console": true,
+    "no-construct": true,
+    "no-debugger": true,
+    "no-duplicate-super": true,
+    "no-duplicate-variable": true,
+    "no-empty": true,
+    "no-eval": true,
+    "no-floating-promises": true,
+    "no-for-in-array": true,
+    "no-inferred-empty-object-type": true,
+    // "no-invalid-this": Won't this be deprecated?
+    "no-misused-new": true,
+    "no-null-keyword": true,
+    "no-shadowed-variable": true,
+    "no-string-literal": true,
+    "no-string-throw": true,
+    // "no-switch-case-fall-through": Won't this be deprecated?
+    "no-unbound-method": true,
+    "no-unsafe-any": true,
+    "no-unsafe-finally": true,
+    "no-unused-expression": true,
+    // "no-unused-variable": Won't this be deprecated?
```

Comment:
still keeping it around; you probably ought to enable it here actually, given the kind of semantics you're going for for `tslint:all`